libretroshare/src/gxs/rsgxsnetservice.cc

```diff
@@ -744,7 +745,7 @@ void RsGxsNetService::syncWithPeers()
 #endif
 }
 
-void RsGxsNetService::generic_sendItem(RsNxsItem *si)
+void RsGxsNetService::generic_sendItem(rs_owner_ptr<RsItem> si)
```

Comment:
Only you know what that template means. A brief comment saying that the method takes ownership of the item would be far more helpful.